index.html

```diff
@@ -14,6 +14,22 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <nav id="mobile-nav">
+        <div id="close-button-container">
+          <button type="button" class="nav-close">
+            <i class="fas fa-times"></i>
+          </button>
+        </div>
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
```

Comment:
- [ ] According to what we have on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=mwBAbz2zTBZMi9fH-0). The message icon should be hidden when we click on the hamburger menu bar. I will suggest we stick to the Figma design template as seen here 👇 
![save](https://user-images.githubusercontent.com/78634841/203851579-58192905-ae10-4e8c-acfc-6643627e5604.png)

compare to what we have in your project

![ham](https://user-images.githubusercontent.com/78634841/203851776-357dcde2-f183-49f7-b238-707278851022.png)